crates/forge/tests/it/cheats.rs

```diff
@@ -26,6 +27,7 @@ async fn test_cheats_local(test_data: &ForgeTestData) {
     }
 
     let mut config = test_data.config.clone();
+    config.fuzz.seed = Some(U256::from(100));
```

Comment:
should this be set in test_data?

Reply:
good point, not sure we should be adding a new `TEST_DATA_WITH_SEED` (we don't have one for isolate either) but default tests shouldn't run with seed - since there's no option to set it inline I've added a test that runs contracts matching `WithSeed` (excluded from defaults), hope this makes sense
pls check https://github.com/foundry-rs/foundry/pull/8807/commits/7e69723609d13ceb3ce744c0f7bbdd3e156f1296